src/cryptography/hazmat/primitives/padding.py

```diff
@@ -122,3 +126,53 @@ def finalize(self):
         res = self._buffer[:-pad_size]
         self._buffer = None
         return res
+
+
+class PKCS7(_BytePadding):
```

Comment:
This project has a general policy against subclassing for a variety of reasons. While it is simple here, we'd prefer the logic to be self-contained. Sorry about that.